packages/terra-arrange/src/terra-dev-site/doc/example/ArrangeAlignment.jsx

```diff
@@ -110,6 +110,7 @@ class ArrangeAlignment extends React.Component {
             {...(this.state.alignFitStart && { alignFitStart: this.state.alignFitStart })}
             {...(this.state.alignFill && { alignFill: this.state.alignFill })}
             {...(this.state.alignFitEnd && { alignFitEnd: this.state.alignFitEnd })}
+            style={{ height: '100%' }}
```

Comment:
Why are you adding this?

Reply:
It may not be necessary any more. Initially, the examples didn't actually work, because the container would scale to whichever was the larger of the icons, so alignment of one icon would work, and the other would not... So this essentially forced the height of the whole arrange container to fill the wrapper.

However, this will likely be gone soon, since I'm overhauling the examples anyway.

Reply:
This is removed in the current deployment via all the shuffling of examples. :)